cmd/krew/cmd/version.go

```diff
@@ -46,6 +47,7 @@ Remarks:
 			{"IndexPath", paths.IndexPath()},
 			{"InstallPath", paths.InstallPath()},
 			{"BinPath", paths.BinPath()},
+			{"OS/Arch", installation.OSArch().String()},
```

Comment:
you can probably just say DetectedPlatform:

Reply:
Done. Thanks.